mars/dataframe/missing/checkna.py

```diff
@@ -57,8 +61,13 @@ def __call__(self, df):
             self.output_types = [OutputType.dataframe]
         elif isinstance(df, SERIES_TYPE):
             self.output_types = [OutputType.series]
-        else:
+        elif isinstance(df, TENSOR_TYPE) or isinstance(df, INDEX_TYPE):
```

Comment:
It's better to return value directly in `isna` for those scalars, no need to create the operand.